Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Create_Address_Object.yml

```diff
@@ -457,12 +457,12 @@ inputs:
   playbookInputQuery:
 - key: Type
   value: {}
-  required: false
+  required: true
```

Comment:
You decided to put it as mandatory, I think we better verify the value with a conditional task (for example add it with the IP provided - Do the IPs exist? task)

Reply:
Done